cmd/post-github/main.go

```diff
@@ -59,7 +54,7 @@ func runGH(
 
 Stress build found a failed test:
 
-%s`, sha, makeCodeLike(test.Message))
```

Comment:
You can't escape the triple-backticks, but there's another way to make a code block: indent every line by four spaces.